subprojects/testing-jvm/src/integTest/resources/org/gradle/testing/junit/JUnitIntegrationTest/junit3Tests/src/test/java/org/gradle/Junit3Test.java

```diff
@@ -5,5 +5,6 @@
 public class Junit3Test extends TestCase {
     public void testRenamesItself() {
         setName("a test that renames itself");
+        fail("epic");
```

Comment:
😆 

Why the change though?

Reply:
This is really an edge case and I wanted to make sure that we at least consistently report a failed test in both JUnit 4 and Vintage.